package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "authmosphere",
-  "version": "1.1.3",
+  "version": "1.1.4",
```

Comment:
issue is tagged with breaking

Reply:
If we sum up some breaking changes we could use something like x.y.z-beta

Reply:
Was just a stub until we decide if that is the 2.0.0 - as we have still the breaking typings PR open